specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,1955 @@
+                            "description": "The security solutions' image offer"
+                        },
+                        "publisher": {
+                            "type": "string",
+                            "description": "The security solutions' image publisher"
+                        },
+                        "sku": {
+                            "type": "string",
+                            "description": "The security solutions' image sku"
+                        }
+                    }
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Type of the resource",
+                    "enum": [
+                        "Microsoft.Security/locations/discoveredSecuritySolution"
+                    ]
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            },
+            {
+                "$ref": "#/definitions/Location"
+            }]
+        },
+        "securityFamily": {
+            "type": "string",
+            "description": "The security family of the discovered solution",
+            "enum": [
+                "Waf",
+                "Ngfw",
+                "SaasWaf",
+                "Va"
+            ]
+        },
+        "JitNetworkAccessPoliciesList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/JitNetworkAccessPolicy"
+                    }
+                }
+            }
+        },
+        "JitNetworkAccessPolicy": {
+            "type": "object",
+            "required": [
+                "kind",
+                "properties",
+                "id",
+                "name",
+                "type",
+                "location"
+            ],
+            "properties": {
+                "properties": {
+                    "required": [
+                        "provisioningState",
+                        "virtualMachines"
+                    ],
+                    "type": "object",
+                    "properties": {
+                        "virtualMachines": {
+                            "type": "array",
+                            "description": "Configurations for Microsoft.Compute/virtualMachine resource type.",
+                            "items": {
+                                "$ref": "#/definitions/JitNetworkAccessRequest"
+                            }
+                        },
+                        "requests": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/JitNetworkAccessPolicyVirtualMachine"
+                            }
+                        },
+                        "provisioningState": {
+                            "type": "string",

```

Comment:
There is no point in making this a required property. No body can update the value of this property. This should actually be marked as `"readOnly": true` as this will only be returned by the server in the response.